packages/lib/src/formatters/formatters.test.ts

```diff
@@ -0,0 +1,53 @@
+import { describe, test } from 'node:test';
+import { equal } from 'node:assert/strict';
+import { ConfigSchema, EvalCommand, InMemoryConfigStore, ConfigSet, ExportCommand } from '@configu/ts';
+import { formatConfigs } from '.';
+
+describe('Formatters', () => {
+  const getEvalResult = async () =>
+    new EvalCommand({
+      store: new InMemoryConfigStore(),
+      set: new ConfigSet('test'),
+      schema: new ConfigSchema('mutate', {
+        KEY0: {
+          type: 'String',
+        },
+        KEY1: {
+          type: 'String',
+        },
+      }),
+      configs: {
+        KEY0: 'KEY0',
+        KEY1: 'KEY1',
+      },
+    }).run();
+
```

Comment:
I made some tests for basic formats.
I decided to put tests inside `lib` dir where tested feature "lives" - I see most tests are located inside `ts` dir though. 

I don't see any existing test cases for other formatters, so I guess I was supposed to create them.